ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -125,29 +204,118 @@ def install_storage_cluster(self):
+            self.select_checkbox_status(
+                status=True, locator=self.dep_loc["wide_encryption"]
+            )
+
+        if self.is_class_encryption:
+            logger.info("Storage class encryption")
+            self.select_checkbox_status(
+                status=True, locator=self.dep_loc["class_encryption"]
+            )
+
+    def configure_kms(self):
+        """
+        Configure KMS
 
-        logger.info("Select Storage Class")
-        self.do_click(deployment["Storage Class Dropdown"], type=By.CSS_SELECTOR)
-        self.do_click(deployment["thin"], type=By.CSS_SELECTOR)
+        """
+        if self.is_use_kms:
+            logger.info(f"kms service name: {self.service_name}")
+            self.do_send_keys(
+                text=self.service_name, locator=self.dep_loc["kms_service_name"]
+            )
+
+            logger.info(f"kms address: {self.kms_address}")
+            self.do_send_keys(
+                text=self.kms_address, locator=self.dep_loc["kms_address"]
+            )
+
+            logger.info(f"kms address port: {self.kms_address_port}")
+            self.do_send_keys(
+                text=self.kms_address_port, locator=self.dep_loc["kms_address_port"]
+            )
+
+            logger.info(f"kms_token: {self.kms_token}")
+            self.do_send_keys(text=self.kms_token, locator=self.dep_loc["kms_token"])
+
+    def verify_ocs_installation(self, timeout_install=300, sleep=20):
+        """
+        Verify OCS Installation
 
-        logger.info("Choose OSD Size")
-        self.do_click(deployment["OSD Size Dropdown"], type=By.CSS_SELECTOR)
-        self.do_click(deployment[self.osd_size], type=By.XPATH)
+        timeout_install (int): Time in seconds to wait
+        sleep (int): Sampling time in seconds
 
-        if self.is_encryption:
-            self.do_click(deployment["Enable Encryption"], type=By.XPATH)
+        """
+        self.navigate_installed_operators_page()
 
-        logger.info("Create Storage Cluster Page")
-        self.do_click(deployment["Create Storage Cluster Page"], type=By.XPATH)
+        self.do_send_keys(
+            locator=self.dep_loc["search_ocs_install"],
+            text="OpenShift Container Storage",
+        )
+        sample = TimeoutSampler(
+            timeout=timeout_install,
+            sleep=sleep,
+            func=self.check_element_text,
+            expected_text="Succeeded",
+        )
+        if not sample.wait_for_func_status(result=True):
+            logger.error(
+                f"OCS Installation status is not Succeeded after {timeout_install} seconds"
+            )
+            raise TimeoutExpiredError
 
-    def install_lso_cluster(self):
+    def install_ocs_ui(self):
         """
-        Install LSO Cluster
+        Install OCS via UI
 
         """
-        pass
+        self.create_catalog_source_yaml()
+        self.install_ocs_operator()
+        self.verify_ocs_installation()
```

Comment:
shouldn't verify_ocs_installation() be called after install_storage_cluster()?

Reply:
On this method, I verify that ocs operator move `succeeded` state , (before create storage cluster) 
![image](https://user-images.githubusercontent.com/61982127/106892834-51c7c780-66f5-11eb-86c5-9307651caa62.png)

Reply:
In that case, I think that the method name should be changed from verify_ocs_installation() to verify_ocs_operator_succeeded()